charts/datadog/templates/_container-fips-proxy.yaml

```diff
@@ -16,6 +16,16 @@
   resources:
 {{ toYaml .Values.fips.resources | indent 4 }}
     # TODO Add config to monitor journald
+{{- if eq (include "should-mount-fips-configmap" .) "true"}}  
+  volumeMounts:
+    - name: fips-config
+      mountPath: /etc/datadog-fips-proxy
+      readOnly: false # Need RW to mount to config path
+    - name: fips-proxy-cfg
+      mountPath: /etc/datadog-fips-proxy/datadog-fips-proxy.cfg
+      subPath: datadog-fips-proxy.cfg
+      readOnly: true
```

Comment:
if we need to mount the same volumes in several container, maybe it is is better to create a function helper? 

Also, do we need to consider window host? I think the current change will break windows deployment if `should-mount-fips-configmap == true`

What I suggest it to create file like https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/_container-host-release-volumemounts.yaml
do manage the `volumeMounts` for fips configmap.

Reply:
I think we'll be okay in a windows deployment because `"should-mount-fips-configmap"` includes the check for `"should-enable-fips"`, which returns `false` if the targetSystem is not linux: https://github.com/DataDog/helm-charts/pull/1095/files#diff-483dad0ace7ed469efb9d93ba2f3d3c2045bbc1cd317f49e7e0a122bdf9d6116R321-R329

Good idea to create a function helper.